packages/labs/virtualizer/src/layouts/shared/SizeCache.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
```

Comment:
The SizeCache file was added in this commit: https://github.com/PolymerLabs/uni-virtualizer/commit/c1b6c56b517a33f1af2e207cfbab6cb64b96ba58#diff-8f89c300b7e83bcbf8413f790b77a41ab4a91f901b855bd18b7a51e6b79a0196

The copyright year should be the year it was created - thus `2021`.

Reply:
Thanks, @AndrewJakubowicz - didn't realize we were to backdate license headers to file creation date.  Seems fair in this case anyways.  Done.